samples/quickstart_installed_oauth2.js

```diff
@@ -32,13 +32,15 @@ const url = require('url');
 const open = require('open');
 const destroyer = require('server-destroy');
 
-// Reads the secrets from a `keys.json` file, which should be downloaded from
-// the Google Developers Console and saved in the same directory with the sample
-// app.
+// Reads the secrets from a `oauth2.keys.json` file, which should be downloaded
```

Comment:
if we move this into a file called `quickstart.js` then it will be automatically included in the README as an example.

Reply:
Thanks for the tip! Moved quickstart_service_account.js to quickstart.js,